Sources/OpenTelemetrySdk/Metrics/Stable/RegisteredReader.swift

```diff
@@ -5,17 +5,17 @@
 
 import Foundation
 import OpenTelemetryApi
-
+import Atomics
```

Comment:
we removed atomics from the framework due to compatibility issues with XCFrameworks iirc.